sharding-core/sharding-core-common/src/main/java/org/apache/shardingsphere/core/strategy/keygen/SnowflakeShardingKeyGenerator.java

```diff
@@ -134,7 +136,13 @@ private long getWorkerId() {
         Preconditions.checkArgument(result >= 0L && result < WORKER_ID_MAX_VALUE);
         return result;
     }
-    
+
+    private long getCustomVibrationValue() {
+        long result = Long.valueOf(properties.getProperty("custom.vibration.value", String.valueOf(VIBRATION_VALUE)));
```

Comment:
should need change other name of "custom.vibration.value", maybe "max.vibration.offset" is better